sdk/src/Services/DynamoDBv2/Custom/DataModel/Context.cs

```diff
@@ -226,6 +226,30 @@ public BatchWrite<T> CreateBatchWrite<T>(DynamoDBOperationConfig operationConfig
             return new BatchWrite<T>(this, config);
         }
 
+        /// <summary>
```

Comment:
Can you update the comments on `BatchWrite<object>` methods to include details about the object typeparam and how it is different from the generic version.